src/Common/src/TypeSystem/IL/Stubs/PInvokeILEmitter.Mangling.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Nit: Instead of putting this on PInvokeILEmitter.Mangling.cs would it make sense to put the full definition of PInvokeTargetNativeMethod  in its own file PInvokeTargetNativeMethod .cs  since this has nothing to do PInvokeILEmitter?